src/static/legacy-header.css

```diff
@@ -1,120 +1,188 @@
+.account-dropdown {
+  height: 52px;
+  margin-right: 6px;
+  margin-top: 4px;
+  padding-top: 2px;
+}
+
+.account-dropdown .dropdown-menu {
+  left: auto;
+  right: 0px;
+  top: 59px;
+}
+
+.account-dropdown .dropdown-menu > li > a > .label {
+  position: relative;
+  top: -1px;
+}
+
+.account-dropdown .dropdown-menu > li > a .fa {
+  left: 0px;
+  position: relative;
+  text-align: left;
+  top: 1px;
+}
+
+.account-dropdown .menu-item {
+  cursor: pointer;
+  height: 29px;
+}
+
+.account-dropdown-signin {
+  margin: 3px 0px;
+  padding: 10px 14px;
+  text-align: center;
+}
+
+.account-dropdown-signin:hover {
+  background: #f5f5f5;
+}
+
+.divider {
+  background-color: #e5e5e5;
+  height: 1px;
+  margin: 9px 0;
+  overflow: hidden;
+}
+
 .dropdown-menu {
-  position: absolute;
-  top: 55px;
-  left: 5px;
-  z-index: 1000;
-  min-width: 160px;
-  padding: 5px 0;
-  margin: 2px 0 0;
-  font-size: 14px;
-  list-style: none;
-  background-color: #fff;
   background-clip: padding-box;
-  border: 1px solid #ccc;
-  border: 1px solid rgba(0, 0, 0, 0.15);
+  background-color: #fff;
   border-radius: 4px;
-  -webkit-box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
+  border: 1px solid #ccc;
   box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
+  font-size: 14px;
+  left: 0;
+  list-style: none;
+  margin-top: 6px;
+  margin: 2px 0 0;
+  min-width: 160px;
+  padding: 5px 0;
+  position: absolute;
+  top: 55px;
+  z-index: 1000;
```

Comment:
z-index is always a fun time, and the source of many bugs in my experience. It doesn't look like setting the z-index is necessary to get the desired behavior here, so I would omit it, unless there was a specific case it was added for.
```suggestion

```